Dockerfile

```
@@ -3,7 +3,7 @@ FROM node:22-bullseye as node_builder
 WORKDIR /app
 COPY . .
 
-RUN npm install
+RUN npm install --production
```

```suggestion
RUN npm install --omit=dev
```
Looks like `--production` [was the old name for the flag](https://stackoverflow.com/a/9276112/10478115)